cmd/promtool/main.go

```diff
@@ -420,6 +427,10 @@ func main() {
 
 	case tsdbDumpCmd.FullCommand():
 		os.Exit(checkErr(dumpSamples(ctx, *dumpPath, *dumpSandboxDirRoot, *dumpMinTime, *dumpMaxTime, *dumpMatch, formatSeriesSet)))
+
+	case tsdbDumpSeriesCmd.FullCommand():
```

Comment:
As I commented, combine with above, fine to just have a simple `switch` on the `--format` flag here I think.